aptos-move/aptos-transactional-test-harness/src/aptos_test_harness.rs

```diff
@@ -0,0 +1,916 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, format_err, Result};
+use aptos_crypto::{
+    ed25519::{Ed25519PrivateKey, Ed25519PublicKey},
+    hash::HashValue,
+    ValidCryptoMaterialStringExt,
+};
+use aptos_state_view::StateView;
+use aptos_types::{
+    access_path::AccessPath,
+    account_config::{aptos_root_address, AccountResource},
+    block_metadata::BlockMetadata,
+    chain_id::ChainId,
+    contract_event::ContractEvent,
+    state_store::state_key::StateKey,
+    transaction::{
+        ExecutionStatus, Module as TransactionModule, RawTransaction, Script as TransactionScript,
+        ScriptFunction as TransactionScriptFunction, Transaction, TransactionOutput,
+        TransactionStatus,
+    },
+    utility_coin::TEST_COIN_TYPE,
+};
+use aptos_vm::{
+    data_cache::{IntoMoveResolver, RemoteStorageOwned},
+    AptosVM,
+};
+use clap::StructOpt;
+use language_e2e_tests::data_store::{FakeDataStore, GENESIS_CHANGE_SET_FRESH};
+use move_deps::{
+    move_binary_format::file_format::{CompiledModule, CompiledScript},
+    move_command_line_common::{
+        address::ParsedAddress, files::verify_and_create_named_address_mapping,
+    },
+    move_compiler::{self, shared::PackagePaths, FullyCompiledProgram},
+    move_core_types::{
+        account_address::AccountAddress,
+        gas_schedule::{GasAlgebra, GasConstants},
+        identifier::{IdentStr, Identifier},
+        language_storage::{ModuleId, ResourceKey, StructTag, TypeTag},
+        move_resource::MoveStructType,
+        transaction_argument::{convert_txn_args, TransactionArgument},
+        value::{MoveTypeLayout, MoveValue},
+    },
+    move_resource_viewer::{AnnotatedMoveValue, MoveValueAnnotator},
+    move_transactional_test_runner::{
```

Comment:
@vgao1996 should we directly fork move_transactional_test_runner to Aptos Repo. 
For example. Our test should rely on Aptos Adapter instead of movetest adapter which is not aware aptos specific changes.

Reply:
Good point. I also want to customize InitCommand and move quickly.